resources/views/student-council/communication-committee/edit.blade.php

```diff
@@ -0,0 +1,198 @@
+@extends('layouts.app')
+
+@section('title')
+<a href="#!" class="breadcrumb">@lang('role.student-council')</a>
+<a href="#!" class="breadcrumb" >@lang('role.communication-committee')</a>
+<a href="{{ route('epistola') }}" style="cursor: pointer" class="breadcrumb">Epistola Collegii</a>
+<a href="#!" class="breadcrumb">Szerkesztés</a>
+@endsection
+@section('student_council_module') active @endsection
+
+@section('content')
+
+<div class="row">
+    <div class="col s12 l8 push-l2">
+        <div class="card">
+            <div class="card-content">
+                <div class="card-title">{{ $epistola->title ?? "Új hír"}}</div>
+                <form method="POST" action="{{ route('epistola.update_or_create') }}" enctype="multipart/form-data">
+                    @csrf
+                    @if($epistola)
+                    <input type="hidden" name="id" value="{{$epistola->id}}">
+                    @endif
+                    <div class="row">
+                        <div class="input-field col s12">
+                            <input id="title" name="title" type="text" class="validate @error('title') invalid @enderror" required
```

Comment:
Just came to me, we use these input-label-error so often, we could add a template for it. I also find myself often using a lot of copy paste when playing with such UI changes.

Reply:
Yes, we should. I wanted to add one for the whole input field, but that needed a lot of variables.